x-pack/plugins/security_solution/common/api/detection_engine/prebuilt_rules/get_prebuilt_rules_and_timelines_status/get_prebuilt_rules_and_timelines_status_route.schema.yaml

```diff
@@ -5,6 +5,9 @@ info:
 paths:
   /api/detection_engine/rules/prepackaged/_status:
     get:
+      x-labels:
+        - ess
+        - serverless
```

Comment:
We will need to add labels to many more endpoints in Security. Should we do it in a separate PR, and remove the labelling in this one?

We can leave changes in a couple endpoints to make the `yarn openapi:bundle` script work.